data/examples/shaders/transition/luma-burn.effect

```diff
@@ -49,6 +49,11 @@ uniform float LumaExponent<
 	float scale = .01;
 > = 150.;
 
+uniform float4 TransitionColor<
+string name = "Color";
+string field_type = "input";
+> = { .8, 0.33, 0.33, 0.75 };
```

Comment:
Default value should be neutral.

Reply:
updating.
- splitting into new techniques, I am impressed how your system is handing the options in the techniques! 
- leaving as non-neutral for dramatic effect in the new techniques
- updated implementation to be the same as sliding bars color

question: Would you preferred 4 floats that can be labelled red, green, blue, alpha or if the labeling of the float4 variables is possible? You have enums for ints, so maybe I am missing the labeling here.

will rebase

Reply:
Labeling for multiple values isn't possible yet, the OBS parser crashes when nesting annotations. Might be possible in the future with a custom parser, but right now it's not possible.